tests/test_jax_nonbonded.py

```diff
@@ -186,31 +190,8 @@ def generate_random_inputs(n_atoms, dim, instance_flags=difficult_instance_flags
     beta = 2.0
     if instance_flags["randomize_beta"]:
         beta += rand()
-    cutoff = 1.2
-    if instance_flags["randomize_cutoff"]:
-        cutoff += rand()
-
-    lambda_plane_idxs = np.zeros(n_atoms, dtype=int)
-    lambda_offset_idxs = np.zeros(n_atoms, dtype=int)
-
-    if instance_flags["randomize_lambda_plane_idxs"]:
-        lambda_plane_idxs = randint(low=-2, high=2, size=n_atoms)
-
-    if instance_flags["randomize_lambda_offset_idxs"]:
-        lambda_offset_idxs = randint(low=-2, high=2, size=n_atoms)
 
-    args = (
-        conf,
-        params,
-        box,
-        lamb,
-        charge_rescale_mask,
-        lj_rescale_mask,
-        beta,
-        cutoff,
-        lambda_plane_idxs,
-        lambda_offset_idxs,
-    )
+    args = (conf, params, box, lamb, charge_rescale_mask, lj_rescale_mask, beta, cutoff)
```

Comment:
Looks like lambda is still used in this file, is there something about the references that still require lambda?

Reply:
The references should be independent of lambda now. But my plan was to hold off on removing lambda from the reference signatures until we do the same for the custom ops potentials (which will be done in the final PR of this series)